dbconn/dbconn_test.go

```diff
@@ -204,6 +204,23 @@ var _ = Describe("dbconn/dbconn tests", func() {
 			Expect(testRecord.Schemaname).To(Equal("schema1"))
 			Expect(testRecord.Tablename).To(Equal("table1"))
 		})
+		It("executes a GET with argument outside of a transaction", func() {
+			arg1 := "1"
+			arg2 := 2
+			two_col_single_row := sqlmock.NewRows([]string{"schemaname", "tablename"}).
+				AddRow("schema1", "table1")
+			mock.ExpectQuery("SELECT (.*)").WithArgs(arg1, arg2).WillReturnRows(two_col_single_row)
+
+			testRecord := struct {
+				Schemaname string
+				Tablename  string
+			}{}
+
+			err := connection.GetWithArgs(&testRecord, "SELECT schemaname, tablename FROM two_columns ORDER BY schemaname", arg1, arg2)
```

Comment:
This doesn't appear to actually use the args you pass in. It would be nice to see a test case that does.